### DISCUSSION TITLE
Feature Request - Permit the models via tags to split the voices to be used.

### DISCUSSION THREAD

**Discussion by Urammar** [erew123/alltalk_tts#58] [Dev Discussions]
Having multiple characters speaking would currently require manual changing of the voice before generation.

Allow [tags] the model can be made aware of to select the next voice. [Fry.wav] Stephen: Because this would be pretty cool.

**Comment by erew123** [comment]
That should in theory be possible...... Ill bop this over to the discussions area, as Im keeping track of things there to implement.

**Comment by erew123** [comment]
@Urammar I'm closing this discussion, but I'm not closing it closing it. I'm actually putting up a central list of feature requests and I am linking back to all requests off that list. So your request will be noted in here https://github.com/erew123/alltalk_tts/discussions/74. I'm also asking on the Feature Requests list that people join in the original thread discussions by following the links.

Thanks

**Comment by crandev** [comment]
Just started using alltalk and this feature would be realy appreciated! ^_^

I tried to work around with making group chats, so that I can set a different voice for every character in sillytavern. But it would be much easier to get a single storyteller ai to set the correct flags for different voices. 

Thanks for your great work!

**Comment by ariadesu** [comment]
Would it be possible to extend this feature (when it's implemented) to also allow specifying a language? It's useful for multilingual sentences.
